cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -88,6 +88,26 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
             }
           });
         });
+
+        it('should have a visible play button and duration', () => {
+          cy.window().then(win => {
+            const media = getBlockData('video', win.SIMORGH_DATA.pageData);
+            if (media) {
+              const {
+                durationISO8601,
+              } = media.model.blocks[1].model.blocks[0].model.versions[0];
```

Comment:
Can I suggest we separate this path to improve readability? E.G.:
```
const aresMediaBlocks = media.model.blocks[1].model.blocks[0];
const { durationISO8601 } = aresMediaBlocks.model.versions[0];
```

Reply:
Agreed. This makes for easier reading. Will implement.